.github/workflows/ci.yml

```diff
@@ -115,6 +115,14 @@ jobs:
           source .venv/bin/activate
           pip install -r tests/matmul/requirements.txt
 
+      - name : E2E comparison of AIE to llvm-cpu
```

Comment:
Oops I moved this before the matmul tests just to make my CI testing faster, will move back to where it was to reduce noise.